src/AisParser/Payload.cs

```diff
@@ -15,9 +15,9 @@ public Payload(string rawValue)
             MessageType = ReadEnum<AisMessageType>(0, 6);
         }
 
-        public string RawValue { get; }
+        public string RawValue; // { get; }
 
-        public AisMessageType MessageType { get; }
+        public AisMessageType MessageType; // { get; }
```

Comment:
Remove commented out code

Reply:
Same as above, MessageType gets set now too, so the get restriction was removed. (Bad modification on my part, I should have just removed it entirely instead of commenting it out)